Test/server/counterexample_commandline.dfy.expect

```diff
@@ -0,0 +1,25 @@
+counterexample_commandline.dfy(29,20): Error: A postcondition might not hold on this return path.
+counterexample_commandline.dfy(17,22): Related location: This is the postcondition that might not hold.
+
+Dafny program verifier finished with 1 verified, 1 error
+Counterexample for first failing assertion: 
```

Comment:
I am not sure it is possible to do entwine the two. To extract a counterexample, one needs the prover model, which, as far as I understand, Boogie prints after everything else. Dafny error trace, on the other hand is created by extending Boogie's `ConsolePrinter ` and overriding `ReportBplError` method. So my hunch is that you would need to make changes to the way and order in which Boogie reports everything to make it so that the counterexample appears right after the error trace.